patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -1,6 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/entity/EntityLivingBase.java
 +++ ../src-work/minecraft/net/minecraft/entity/EntityLivingBase.java
-@@ -201,10 +201,11 @@
+@@ -72,6 +72,7 @@
+ import net.minecraft.util.math.Vec3d;
+ import net.minecraft.world.World;
+ import net.minecraft.world.WorldServer;
++import net.minecraftforge.common.ForgeHooks;
```

Comment:
Do not use imports in patches, instead use the fully qualified name where it is used.

Reply:
Fixed in latest commit.